mlir/Maintainers.md

```diff
@@ -44,6 +44,39 @@ MLIR components pertaining to egress flows from MLIR, in particular to LLVM IR.
   [@gysit](https://github.com/gysit) (GitHub),
   gysit (Discourse)
 
+### Dialects
+
+The `egress` maintainer refers to the people working in the Egress category,
+with the point-of-contact being the maintainers above. Named maintainers, if
+available, should be contacted first, as they're more active in those areas.
+
+#### Lowering Dialects
+* ‘llvm’ Dialect (egress)
```

Comment:
Should we use normal single quotes / backticks? I think these are some apple smart quotes ones?

Reply:
I just wanted to be consistent with https://github.com/llvm/llvm-project/pull/150945. @rengolin , what's your preference?